### DISCUSSION TITLE
Discussions are where issues go to die

### DISCUSSION THREAD

**Discussion by mixonomicon** [community/community#3858] [Discussions]
Maintainers are actively using discussions to avoid dealing with issues. Prove me wrong.

**Comment by TonyGravagno** [comment]
We've been using Issues for Q&A for years : Issues are where discussions go to die.
Each has it's place. I just posted a Question, in Discussions. It was not an Issue. It got an Answer. That's what I needed.
I welcome and encourage use of this feature.

**Comment by hugalafutro** [comment]
Discussions are where I can ask question without authors closing the issue with not-a-bug.
